python/tvm/script/tir/__init__.pyi

```diff
@@ -0,0 +1,239 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=redefined-builtin
+from typing import (
+    Any,
+    Callable,
+    ContextManager,
+    Dict,
+    Iterable,
+    Object,
+    Optional,
+    Tuple,
+    Union,
+    Sequence,
+    List,
+    Mapping,
+)
+from tvm.tir.function import PrimFunc
+from tvm.tir import PrimExpr, Buffer, IterVar, Var
+from .node import BufferSlice
+
+"""
+Variables and constants
+"""
+
+def bool(imm: int) -> PrimExpr: ...
```

Comment:
imm should has type Union[PrimExpr, bool, Number]

Reply:
the definition of `bool` here actually shadows all use of native `bool` type annotation of Python in this file. My workaround is to not include type annotation for method/class parameter that may be typed `bool`. E.g. this line would just be `def bool(imm) -> PrimExpr: ...`